src/services/Store/store/helpers.ts

```diff
@@ -102,3 +104,28 @@ export const canImport = (toImport: Partial<LocalStorage>, store: LocalStorage)
     return diff.length === 0;
   }
 };
+
+export const destructureCoinGeckoIds = (
+  rates: IRates,
+  coinGeckoIdMapping: Record<string, string>
+): IRates => {
+  // From: { ["ethereum"]: { "usd": 123.45,"eur": 234.56 } }
+  // To: { [uuid for coinGeckoId "ethereum"]: { "usd": 123.45, "eur": 234.56 } }
+  const updateRateObj = (acc: any, curValue: TTicker): IRates =>
+    Object.entries(coinGeckoIdMapping).reduce((acc, [assetUuid, coinGeckoId]) => {
```

Comment:
Will there be conflicts re-using `acc` ? I wasn't sure, so i just used another var name.

Reply:
Oh yeah nice catch, typescript didn't complained about it and it didn't blew up so I didn't saw it. Will take a look at it tomorrow